### PR TITLE
Git - What Is Stored in a Commit?

### PR DESCRIPTION
**Pull request by kapnobatai137**
Add footnote and resource link on checksums

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/git/essentials/commits/what-is-stored-in-a-commit.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/git/essentials/commits/what-is-stored-in-a-commit.md?ref=1f3f2b19ed75c81ed75a0b5bad689aca22ede5b2)

File: git/essentials/commits/what-is-stored-in-a-commit.md [hunk 1]
```diff
@@ -39,3 +40,17 @@ The usual commit object contains a hash-code, author’s name and email, date an
...
+
+---
+
+## Footnotes
+
+[1: Checksum]
+
+Checksums are used to check files and other data to confirm they are identical.
+
+A checksum is produced by running the file through an algorithm (e.g. MD5, SHA-1, SHA-256, SHA-512) and it produces a string of fixed length. The size or type of the file doesn't matter, the length of the string will always be the same.
```
Comment (nem035) [inline git/essentials/commits/what-is-stored-in-a-commit.md:52]:
```suggestion
A checksum is produced by running the file through an algorithm (e.g. MD5, SHA-1, SHA-256, SHA-512) and it produces a string of fixed length. The size or type of the file doesn't matter and the length of the string will always be the same.
```

Comment (nem035) [reply git/essentials/commits/what-is-stored-in-a-commit.md:52]: Should we add an example of a checksum maybe? We can generate one for an image file we host?

Comment (kapnobatai137) [reply git/essentials/commits/what-is-stored-in-a-commit.md:52]: good shout, will add an example :+1:

**Review by nem035 — DISMISSED** [review]

**Pull request merged**